conf/app.routes

```diff
@@ -65,8 +65,10 @@ POST        /create-profile/ukims-number/check
 GET         /update-profile/ukims-number                                                    controllers.UkimsNumberController.onPageLoadUpdate
 POST        /update-profile/ukims-number                                                    controllers.UkimsNumberController.onSubmitUpdate
 
-GET         /update-record/:recordId/categorisation/start                                   controllers.CategoryGuidanceController.onPageLoad(recordId: String)
-POST        /update-record/:recordId/categorisation/start                                   controllers.CategoryGuidanceController.onSubmit(recordId: String)
+GET         /update-record/:recordId/categorisation/start                                   controllers.CategoryGuidanceController.onPageLoad(mode: Mode = NormalMode, recordId: String)
+POST        /update-record/:recordId/categorisation/start                                   controllers.CategoryGuidanceController.onSubmit(mode: Mode = NormalMode, recordId: String)
+GET         /update-record/:recordId/categorisation/start/check                                   controllers.CategoryGuidanceController.onPageLoad(mode: Mode = CheckMode, recordId: String)
```

Comment:
I never expect the CheckMode version of the page to be displayed to the user. Again its there just because recategorisation needs to route through it.